focuspoints.lrdevplugin/PentaxDelegates.lua

```diff
@@ -151,6 +153,8 @@ function PentaxDelegates.getAfPointsContrast(photo, metaData)
         })
       end
     end
+  elseif (contrastAfMode[1] == "Face Detect AF" and facesDetected == 0) then
+    LrErrors.throwUserError("Face Detect AF mode enabled, but no faces detected.")
```

Comment:
Sadly, it doesn't. Even though the camera does focus properly when no faces are detected (defaults to auto zone select), the stupid camera does not record this information in the metadata. I'm hoping this is only the case for my camera.